Packs/CyberArkPAS/Integrations/CyberArkPAS/CyberArkPAS.py

```diff
@@ -503,7 +503,7 @@ def verify_credentials(self,
         """
         This function uses the V1 CyberArk PAS api, currently there is no matching function in V2
         """
-        url_suffix = f"/PasswordVault/WebServices/PIMServices.svc/Accounts/{account_id}/VerifyCredentials"
+        url_suffix = f"/PasswordVault/API/Accounts/{account_id}/Verify"
```

Comment:
why this change? are we testing it?

Reply:
I run the old api command and got an error, it looks like it is not supported anymore...
I believe that is some point it will happen to all the old ones too (but the other commands that are using the old api don't have a new version yet but this one has).

Reply:
legit. please add in the readme for all the commands that use the v1(delete/add safe and if there are any others) a note so the customers will now that it is bound to change. you can consult with andrew/shirley to make sure it is written correctly.